python/love/commander/reports.py

```diff
@@ -160,7 +160,8 @@ def act(index, actuator) -> int:
     reports_app.router.add_post("/m1m3-bump-tests", query_m1m3_bump_tests)
     reports_app.router.add_post("/m1m3-bump-tests/", query_m1m3_bump_tests)
 
-    async def on_cleanup():
+    async def on_cleanup(reports_app):
+        # Do cleanup
```

Comment:
Very good practice to add a comment here but please let the comment explain why the method is empty. Something like `This code doesn't require cleaning up.` or so would do.